upgrading/rosa-hcp-upgrading.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="rosa-hcp-upgrading"]
+= Upgrading {hcp-title}
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-upgrading
+
+toc::[]
+
+[id="rosa-hcp-lifecycle-policy_{context}"]
+== Life cycle policies and planning
+
+To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
+
+Upgrades are manually initiated. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+
+[id="rosa-hcp-upgrading-a-cluster"]
+== Upgrading a ROSA cluster
+You can upgrade {hcp-title-first} clusters by using individual upgrades through the `rosa` CLI.
+
+[NOTE]
+====
+When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration.
```

Comment:
based on your workload configuration and (for machinepool upgrades) the number of worker nodes.